blob-storage.tf

```diff
@@ -0,0 +1,83 @@
+provider "azurerm" {
```

Comment:
any reason you're not including the app gateway config?

Reply:
wasn't aware i'll have to include palo in here :). was wondering whether app gateway should go in this pr or together with https://tools.hmcts.net/jira/browse/BPS-870 ticket

Reply:
you can't test it without the app gateway, whether this PR or another, it's up to you.

Reply:
fair enough. i'll include here - all in one :)